### PR TITLE
MRC-1553 Directions Overview, Multi Floor Demo (beta)

### PR DESCRIPTION
**Pull request by vercasson**
- [x] This change is documented in CHANGELOG.md

### CODE REVIEW

File: demo/directions-overview-multifloor/style.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  color: #666;
+}
+
+.directions-overlay strong {
+  color: #98c379;
+}
+/* .directions-overlay p {
+  margin: 10px 0;
+} */
+
```
Comment (amyhliao) [inline demo/directions-overview-multifloor/style.css:38]: Are these needed?

File: demo/directions-overview-multifloor/style.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+.directions-request {
+  line-height: 1.5em;
+}
+
+.directions-request em {
+  color: #d19a66;
+  /* margin-top: 10px; */
```
Comment (amyhliao) [inline demo/directions-overview-multifloor/style.css:45]: This one too

File: demo/directions-overview/style.css [hunk 3]
```diff
@@ -3,71 +3,57 @@ body {
...
 }
 
-.sidebar-container p {
-  padding-left: 10px;
+.directions-overlay strong {
+  color: #98c379;
 }
+/* .directions-overlay p {
+  margin: 10px 0;
+} */
```
Comment (amyhliao) [inline demo/directions-overview/style.css:37]: Another commented out style

**Review by amyhliao — APPROVED** [review]
Looks good!!!

File: demo/directions-overview-multifloor/style.css (hunk 2 above)
Comment (vercasson) [reply demo/directions-overview-multifloor/style.css:45]: thanks

**Review by tarakreft — APPROVED** [review]

**Pull request merged**